src/models.rs

```diff
@@ -63,3 +63,25 @@ pub struct NewApiToken {
     pub token: Vec<u8>,
     pub expires_at: Option<SystemTime>,
 }
+
+#[derive(Queryable, Selectable, Debug, Clone)]
+#[diesel(table_name = crate::schema::uploads)]
+#[diesel(check_for_backend(diesel::pg::Pg))]
+pub struct Upload {
+    pub id: Uuid,
+    pub source_code_ipfs_hash: String,
+    pub forc_version: String,
+    pub abi_ipfs_hash: Option<String>,
+    pub bytecode_identifier: Option<String>,
+    pub created_at: SystemTime,
```

Comment:
`SystemTime` does not support timezones in the DB and is generally less ergonomic than `time::OffsetDateTime`; for persistent data I'd reccomend TZ aware datetime primitive.

Reply:
It looks like OffsetDateTime is only [supported](https://docs.rs/diesel/latest/diesel/sql_types/struct.Timestamp.html) for MySql in diesel :(

The timestamp isn't used for anything other than debugging at the moment, so I've opened #21  to follow up.